src/georaster-layer-for-leaflet.ts

```diff
@@ -164,38 +180,40 @@ const GeoRasterLayer = L.GridLayer.extend({
     const bottomRight = rasterCoordsForTileCoords(numberOfSamplesDown - 1, numberOfSamplesAcross - 1);
 
     const getValuesOptions = {
-      bottom: bottomRight.y,
+      bottom: bottomRight?.y,
```

Comment:
The problem lies with the existing `else` where `rasterCoordsForTileCoords` on line 147 could return `null`. If there are better defaults it could return, then the optional chaining could be removed.